### PR TITLE
[Chore](status) avoid empty error msg on status

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes

Sometime we will meet error status with empty error msg. 
This is because sometimes developers forget to pass specific content.
So I removed the state's constructor for the empty message, and modify the relevant code.

```
errCode = 2, detailMessage = Create rollup replicas failed. Error: 10003: [(172.16.0.60)[E-405]], null, 2592000]
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/common/status.h [hunk 1]
```diff
@@ -351,19 +351,6 @@ class Status {
...
-            status._err_msg = std::make_unique<ErrMsg>();
-            status._err_msg->_stack = get_stack_trace();
-        }
-#endif
-        return status;
-    }
-
     template <typename... Args>
     Status static Error(int code, std::string_view msg, Args&&... args) {
         Status status;
```
Comment (yiguolei) [inline be/src/common/status.h:356]: This  error message lost stacktrace, maybe forget in previous PR.

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**